lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -296,6 +305,65 @@ spec = parallel $ do
                     }
                 )
 
+        it "'migrate' db with new FeePolicy" $
+            testMigrationUpdateFeeValue @ShelleyKey
+                "shelleyRole-v2020-10-13.sqlite"
+                (LinearFee (Quantity 155381.0)  (Quantity 44.0))
+                (Coin 2000000)
+
+        it "'migrate' db to add fees to transactions" $
```

Comment:
@piotr-iohk do you see any other interesting transaction from this database maybe that would be worth controlling?

Reply:
Maybe two more with a bit more inputs / outputs:

`99907bf6ac73f6fe6fe25bd6b68bae6776425b9d15a7c46c7a49b85b8b03f291`:

10 inputs
10 outputs
withdrawal = 565251

`15940a7c1df8696279282046ebdb1ee890d4e9ac3c5d7213f360921648b36666`:

1 input
33 outputs
withdrawal = 561120

Reply:
> Maybe two more with a bit more inputs / outputs:

Good idea!